api/management/commands/loaddata.py

```diff
@@ -2,8 +2,9 @@
 import csv
 
 from django.core.management.base import BaseCommand
+import pandas as pd
```

Comment:
Why are we adding pandas an a dependency in a web app? Using it to just load a CSV for a data model seems like overkill